int16 poseX
int16 poseY
int16 angle